contracts/src/v0.8/automation/chains/OptimismModule.sol

```diff
@@ -16,13 +16,19 @@ contract OptimismModule is ChainModuleBase {
   uint256 private constant FIXED_GAS_OVERHEAD = 60_000;
   uint256 private constant PER_CALLDATA_BYTE_GAS_OVERHEAD = 270;
 
-  function getCurrentL1Fee() external view override returns (uint256) {
-    return OVM_GASPRICEORACLE.getL1Fee(bytes.concat(msg.data, OP_L1_DATA_FEE_PADDING));
+  // @dev This will be updated to use the new function introduced by OP team
+  function getCurrentL1Fee(uint256 dataSize) external view override returns (uint256) {
+    return _getL1Fee(dataSize);
   }
 
   function getMaxL1Fee(uint256 dataSize) external view override returns (uint256) {
+    return _getL1Fee(dataSize);
```

Comment:
Can we combine these two functions since they do the exact same thing? I assume the answer depends on whether we decide to charge actual L1 data usage or max L1 data usage.